packages/ketcher-react/src/script/editor/actions/bond.ts

```diff
@@ -137,25 +151,31 @@ export function fromBondsMerge(restruct, mergeMap) {
   return action
 }
 
-function fromBondFlipping(restruct, id) {
+function fromBondFlipping(restruct: ReStruct, id: number): Action {
   const bond = restruct.molecule.bonds.get(id)
 
   const action = new Action()
   action.addOp(new BondDelete(id))
-  action.addOp(new BondAdd(bond.end, bond.begin, bond)).data.bid = id
+  ;(action.addOp(
+    new BondAdd(bond?.end, bond?.begin, bond)
+  ) as BondAdd).data.bid = id
   // todo: swap atoms stereoLabels and stereoAtoms in fragment
   return action.perform(restruct)
 }
 
-export function fromBondStereoUpdate(restruct, bid, withReverse) {
+export function fromBondStereoUpdate(
+  restruct: ReStruct,
+  bid: number,
+  withReverse?: boolean
+): Action {
   const struct = restruct.molecule
   let bond = struct.bonds.get(bid)
   const action = new Action()
 
-  if (bond.stereo === Bond.PATTERN.STEREO.NONE) {
+  if (bond?.stereo === Bond.PATTERN.STEREO.NONE) {
     const neigs = atomGetNeighbors(restruct, bond.begin)
     const stereoNeig = neigs.find(
-      item => item.bid !== bid && struct.bonds.get(item.bid).stereo > 0
+      item => item.bid !== bid && Number(struct.bonds.get(item.bid)?.stereo) > 0
```

Comment:
No needs to wrap 'stereo' into Number() - it should be marked as 'number' type or 'number?' by default